.travis/run.sh

```diff
@@ -85,6 +85,60 @@ for BOARD in $BOARDS; do
 				PROG=$PROG BOARD=$BOARD TARGET=$TARGET make load || true
 			done
 		fi
+		
+		# Copy built files
+		if [ -z $GITHUB_TOKEN ]; then
+			echo ""
+			echo ""
+			echo ""
+			echo "- No Github token so unable to copy built files"	
+		else
+			echo ""
+			echo ""
+			echo ""
+			echo "- Fetching non shallow to get git version"
+			echo "---------------------------------------------"
+			git fetch --unshallow && git fetch --tags
+			GIT_REVISION=`git describe`
+			echo "============================================="
+			echo ""
+			echo ""
+			echo ""
+			echo "- Copying built files"
+			echo "---------------------------------------------"
+			COPY_REPO_OWNER="sealne"
+			COPY_REPO="HDMI2USB-firmware-prebuilt"
+			COPY_DEST="archive/$GIT_REVISION/$BOARD/$TARGET"
+			rm -rf $COPY_REPO
+			git clone https://$GITHUB_TOKEN@github.com/$COPY_REPO_OWNER/$COPY_REPO.git			
+			cd $COPY_REPO
+			mkdir -p $COPY_DEST
+			# Not currently built so use .bit instead
+			#cp ../third_party/misoc/build/*.xsvf $COPY_DEST
+			cp ../third_party/misoc/build/*.bit $COPY_DEST
+			cp ../build/output.*.log $COPY_DEST
+			# Only hdmi2usb is considered usable just now
+			UNSTABLE_LINK="$BOARD/firmware/unstable"
+			if [ "$TARGET" = "hdmi2usb" ]; then
+				# Copy FX2 firmware
```

Comment:
Can we move updating the symlinks to a script which this calls? Something like;
```bash
# update-symlink.sh <board> <target> <type> <git revision>
update-symlink.sh atlys hdmi2usb unstable v0.0.0-789-abdc123
```

Then we can reuse it for updating the stable / testing links too.




That way we can